pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -78,10 +79,10 @@ const (
 type CheckWrapper struct {
 	ID             string                       `yaml:"id" json:"id"`
 	Text           string                       `json:"d"`
-	Type           string                       `json:"-"`
+	Type           string                       `json:"tt"`
```

Comment:
Where do we set the type?

Reply:
at this line https://github.com/rancher/security-scan/pull/30/files#diff-cdbf6a6affc59d13e5fa37a2e5a29596d90e108f7bf5cc20a29e7d0929d56069R121